sycl/source/detail/device_impl.cpp

```diff
@@ -210,8 +209,7 @@ vector_class<device> device_impl::create_sub_devices(
   const cl_device_partition_property Properties[3] = {
       CL_DEVICE_PARTITION_BY_AFFINITY_DOMAIN,
       (cl_device_partition_property)AffinityDomain, 0};
-  size_t SubDevicesCount =
-      get_info<info::device::partition_max_sub_devices>();
+  size_t SubDevicesCount = get_info<info::device::partition_max_sub_devices>();
```

Comment:
It doesn't look like you've made any changes here and above in this file. Could you please preserve old formatting?

Reply:
This is not me, it is clang-format.   I do not know why it suddenly changed these lines.

Reply:
it is because clang-format was applied to the whole file. Please apply it only to the changes made in this PR.